app/views/layouts/_flash_messages.html.erb

```diff
@@ -0,0 +1,7 @@
+<% flash.each do |name, msg| -%>
+  <div class='flash-holder'>
+    <% if name != 'alert' %>
+      <%= content_tag :p, msg, class: 'govuk-heading-m flash-message' %>
```

Comment:
Could we derive this class name from the flash 'name'?
Worried that this will be hard to find if we do create an alert flash message

Reply:
Yeah, I wasn't sure how to lay this out, felt a bit hokey. 

There's already an error messages partial that Tom put together, I could bring together the two to make things a bit clearer.

I'll take a look at it, but I'm not sure if I should do that as part of a separate PR?